python/cugraph/cugraph/tests/utils.py

```diff
@@ -185,6 +186,12 @@ def create_obj_from_csv(
             directed=(obj_type is cugraph.DiGraph),
             edgevals=edgevals,
         )
+    elif isinstance(obj_type, (cugraph.Graph, cugraph.DiGraph)):
```

Comment:
I don't think `DiGraph` needs to be in this list since inheritance will allow it to be recognized as a `Graph`